bindings/python/unicorn/unicorn.py

```diff
@@ -817,6 +824,8 @@ def hook_add(self, htype: int, callback: UC_HOOK_CALLBACK_TYPE , user_data: Any=
                 cb = ctypes.cast(UC_HOOK_INSN_OUT_CB(self._hook_insn_out_cb), UC_HOOK_INSN_OUT_CB)
             if arg1 in (x86_const.UC_X86_INS_SYSCALL, x86_const.UC_X86_INS_SYSENTER):  # SYSCALL/SYSENTER instruction
                 cb = ctypes.cast(UC_HOOK_INSN_SYSCALL_CB(self._hook_insn_syscall_cb), UC_HOOK_INSN_SYSCALL_CB)
+            if arg1 == x86_const.UC_X86_INS_CPUID:  # CPUID instruction
```

Comment:
See `UC_HOOK_CALLBACK_TYPE` and add this hook to the possible callback types.

Reply:
Added!